modules/contentbox-admin/views/settings/rawSettingsTable.cfm

```diff
@@ -53,7 +53,7 @@
 					</a>
 					
 					<!--- Delete Command --->
-					<a 	class="btn btn-sm btn-danger confirmIt" 
+					<a 	class="btn btn-sm btn-danger #setting.getIsCore() ? '' : 'confirmIt'#"
```

Comment:
In lucee you could write ternary operator in hashes and get executed, not sure in ACF.